src/main/java/net/snowflake/ingest/streaming/internal/SnowflakeStreamingIngestClientInternal.java

```diff
@@ -187,11 +190,29 @@ public class SnowflakeStreamingIngestClientInternal<T> implements SnowflakeStrea
           throw new SFException(e, ErrorCode.KEYPAIR_CREATION_FAILURE);
         }
       } else {
```

Comment:
Should we do a check here that the AUTHORIZATION_TYPE is OAuth, and throw an exception if user gives us something we don't understand?

Reply:
The properties check is [here](https://github.com/snowflakedb/snowflake-ingest-java/blob/e3356bd768a7c01e8176427ef6c6f8f6943d4d67/src/main/java/net/snowflake/ingest/utils/Utils.java#L175C7-L177C100) before passing props to the internal client. Should we also add a check here?

Reply:
no, this looks good, thanks!